src/app/legacy/psammead/psammead-assets/src/svgs.jsx

```diff
@@ -1,55 +1,55 @@
+export { default as BBC_BLOCKS_DARK_MODE } from './svgs/bbcBlocksDarkMode';
+export { default as coreIcons } from './svgs/coreIcons';
+export { default as mediaIcons } from './svgs/mediaIcons';
+export { default as navigationIcons } from './svgs/navigationIcons';
+export { default as afaanoromoo } from './svgs/afaanoromoo';
+export { default as afrique } from './svgs/afrique';
+export { default as amharic } from './svgs/amharic';
+export { default as arabic } from './svgs/arabic';
+export { default as archive } from './svgs/archive';
+export { default as azeri } from './svgs/azeri';
+export { default as bengali } from './svgs/bengali';
+export { default as burmese } from './svgs/burmese';
+export { default as cymrufyw } from './svgs/cymrufyw';
+export { default as gahuza } from './svgs/gahuza';
+export { default as gujarati } from './svgs/gujarati';
+export { default as hausa } from './svgs/hausa';
+export { default as hindi } from './svgs/hindi';
+export { default as igbo } from './svgs/igbo';
+export { default as indonesia } from './svgs/indonesia';
+export { default as japanese } from './svgs/japanese';
+export { default as korean } from './svgs/korean';
+export { default as kyrgyz } from './svgs/kyrgyz';
+export { default as learningenglish } from './svgs/learningenglish';
+export { default as marathi } from './svgs/marathi';
+export { default as mundo } from './svgs/mundo';
+export { default as naidheachdan } from './svgs/naidheachdan';
+export { default as nepali } from './svgs/nepali';
+export { default as news } from './svgs/news';
+export { default as newsround } from './svgs/newsround';
+export { default as pashto } from './svgs/pashto';
+export { default as persian } from './svgs/persian';
+export { default as pidgin } from './svgs/pidgin';
+export { default as portuguese } from './svgs/portuguese';
+export { default as punjabi } from './svgs/punjabi';
+export { default as russian } from './svgs/russian';
+export { default as scotland } from './svgs/scotland';
+export { default as serbian } from './svgs/serbian';
+export { default as sinhala } from './svgs/sinhala';
+export { default as somali } from './svgs/somali';
+export { default as sport } from './svgs/sport';
+export { default as swahili } from './svgs/swahili';
+export { default as tamil } from './svgs/tamil';
+export { default as telugu } from './svgs/telugu';
+export { default as thai } from './svgs/thai';
+export { default as tigrinya } from './svgs/tigrinya';
+export { default as turkce } from './svgs/turkce';
+export { default as urdu } from './svgs/urdu';
+export { default as ukchina } from './svgs/ukchina';
+export { default as ukrainian } from './svgs/ukrainian';
+export { default as uzbek } from './svgs/uzbek';
+export { default as vietnamese } from './svgs/vietnamese';
+export { default as weather } from './svgs/weather';
+export { default as yoruba } from './svgs/yoruba';
+export { default as zhongwen } from './svgs/zhongwen';
```

Comment:
These were angry... In order to properly reexport an export, they needed to be wrapped in `defaults`'s.